model_deployment/README.md

```diff
@@ -17,18 +17,22 @@
 #### 4.2 Setup virtual environment in Windows in CMD
  - Run the following commands in cmd.exe
  ```
-    pip3 install virtualenv 
+    pip3 -m install virtualenv 
     virtualenv venv
     venv\Scripts\activate.bat
  ```
  
  #### 4.3 Setup virtual environment in Windows in PowerShell
   - Run the following commands in PowerShell.exe
  ```
-    pip3 install virtualenv 
+    pip3 -m install virtualenv 
```

Comment:
There is no such command. It should be like this `python3 -m pip3 install virtualenv`.